most-used-words.el

```diff
@@ -50,97 +50,170 @@
 
 (require 'cl-lib)
 
+(defgroup most-used-words nil
+  "Display most used words in buffer."
+  :prefix "most-used-words-"
+  :group 'tool
+  :link '(url-link :tag "Repository" "https://github.com/udyantw/most-used-words"))
+
+(defcustom most-used-words-word-display 3
+  "Default word data to display."
+  :type 'integer
+  :group 'most-used-words)
+
+(defcustom most-used-words-display-type 'buffer
+  "The method to display most used words."
+  :type '(choice (const :tag "buffer" buffer)
+                 (const :tag "table" table))
+  :group 'most-used-words)
+
+(defvar most-used-words--buffer-name "*Most used words*"
+  "Buffer name for displaying most used words.")
+
+(defvar most-used-words--buffer nil
+  "Recorded buffer for currently doing most used words.")
+
+;;; Util
+
 (cl-defmacro most-used-words-with-view-buffer (buffer &body body)
+  "Default BUFFER for this package to display and execute BODY."
```

Comment:
You might want to change the doc string here, I have a temporary one just to process through `checkdoc` command.

Reply:
I just saw that I didn't put a docstring for the macro. Oops.
I think this should do:
"Execute BODY in BUFFER and view it in another window."

Reply:
Resolve conflict, resolve this conversation!